build.rs

```diff
@@ -22,6 +22,13 @@ fn main() {
 
     let _build = cxx_build::bridge("src/bridge.rs");
 
+    let mut config = Config::new(".");
+    let profile = config.get_profile();
+
+    if profile == "Debug" {
+        config.define("SIMDJSON_DEVELOPER_MODE", "ON");
+    }
+
     let dst = Config::new(".").build().join("lib");
```

Comment:
You are configuring CMake twice here.

Reply:
Unfortunately `SIMDJSON_DEVELOPER_MODE` is not what I actually want :upside_down_face: , so I disabled it back.